CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## Next
+- Added support for Snowflake query tags at the connection and model level ([#1030](https://github.com/fishtown-analytics/dbt/issues/1030))
```

Comment:
Can you please move this into the `dbt 0.18.0` section, put it under a `### Features` heading, and make it link to both the issue and this PR?